ui/src/services/dataset.js

```diff
@@ -85,6 +85,20 @@ class DatasetService {
     });
   }
 
+  //Code for displaying associated projects for a particular dataset
+
+  getProjects({ id, params }) {
```

Comment:
Don't catch errors here. There could be cases where we don't want to show errors when projects-retrieval fails.

Move the catch block to the component that calls the getProjects() method.